data/mods/Magiclysm/items/alchemy_items.json

```
@@ -301,5 +301,125 @@
     "bashing": 10,
     "to_hit": 1,
     "category": "spare_parts"
+  },
+  {
+    "id": "alchemist_bomb",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "Мana bomb" },
+    "description": "Mana bombs are powerful explosive devices developed by a little-known alchemist group for use against enemy spellcasters.  These bombs are fueled by pure arcane energy and can kill instantly.  The exact workings are unclear, but they appear to use some kind of massive mana burn.",
+    "weight": "430 g",
+    "volume": "200 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "to_hit": -1,
+    "bashing": 14,
+    "material": [ "steel", "plastic" ],
+    "symbol": "*",
+    "color": "green",
+    "use_action": {
+      "need_wielding": true,
+      "target": "alchemist_bomb_act",
+      "msg": "You mix the contents of the bottle, and it starts to sizzle.",
+      "target_charges": 8,
+      "active": true,
+      "menu_text": "Mix liquid",
+      "type": "transform"
+    },
+    "flags": [ "BOMB", "GRENADE" ]
+  },
+  {
+    "id": "alchemist_bomb_act",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "mana bomb (lit)", "str_pl": "mana bombs (lit)" },
+    "description": "The liquid in the bottle is sizzling.  Throwing it away from you, preferably at an enemy, would be a very good idea.",
+    "weight": "430 g",
+    "volume": "200 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "to_hit": -3,
+    "bashing": 14,
+    "material": [ "paper", "powder" ],
```

```suggestion
    "material": [ "concentrated_mana", "crystallized_mana" ],
```
The crystallized mana is being used up.